app/decorators/ephemera_project_decorator.rb

```diff
@@ -25,4 +29,20 @@ def manageable_structure?
   def title
     super.first
   end
+
+  def slug
+    Array.wrap(super).first
```

Comment:
If we want it to be single-value, we should enforce that on the changeset. But I'm not sure we want it to be single-value. I'm concerned we actually need 2 properties, like current_slug and all_slugs, so that the front-end app can continue routing to old / bookmarked urls. I think this may need more discussion / a new issue.

Reply:
Oh, it already is enforced on the changeset, okay. Well the rest of my concern still stands :)

Reply:
I've created a new issue for handling changed slugs gracefully — not sure if it's ultimately something that needs to change in DPUL or Figgy, but it's something that should work: https://github.com/pulibrary/figgy/issues/395